internal/operators/ocs/validations.go

```diff
@@ -120,19 +120,23 @@ func (o *operator) computeNodeResourceUtil(host *models.Host, nodeInfo *resource
 		return status, err
 	}
 
-	disks := getValidDiskCount(inventory.Disks)
-	if disks > 1 { // OCS must use only the non-boot disks
-		requiredDiskCPU := int64(disks-1) * o.config.OCSRequiredDiskCPUCount
-		requiredDiskRAM := int64(disks-1) * o.config.OCSRequiredDiskRAMGiB
+	disks, err := getValidDiskCount(inventory.Disks)
```

Comment:
Please consider changing the 'disks' variable name to diskCount or something more indicative